uns_builder_test.go

```diff
@@ -26,9 +26,9 @@ func TestUnsBuilder(t *testing.T) {
 
 func TestUnsBuilderSetBackend(t *testing.T) {
 	t.Parallel()
-	backendl1, _ := ethclient.Dial("https://rinkeby.infura.io/v3/c5da69dfac9c4d9d96dd232580d4124e")
-	backendl2, _ := ethclient.Dial("https://matic-testnet-archive-rpc.bwarelabs.com/")
-	builder := NewUnsBuilder().SetEthereumNetwork("rinkeby").SetL2EthereumNetwork("mumbai")
+	backendl1, _ := ethclient.Dial("https://eth-goerli.alchemyapi.io/v2/r3soltxtCL_KhZGl7EwWsWax5ll_MFTN")
+	backendl2, _ := ethclient.Dial("https://polygon-mumbai.g.alchemy.com/v2/EP3SMW2f-2FMABuWuEdHdxKq_v1_ww8")
```

Comment:
Typo in the key `EP3SMW2f-2FMABuWuEdHdxKq_v1_ww82` (missing the ending `2`)

```suggestion
	backendl2, _ := ethclient.Dial("https://polygon-mumbai.g.alchemy.com/v2/EP3SMW2f-2FMABuWuEdHdxKq_v1_ww82")
```